packages/skin/src/routes/_index/component/card/+page.marko

```diff
@@ -0,0 +1,566 @@
+        </div>
+        <h3 class="card__title">Authentic Rookie Cards</h3>
+        <div class="card__description">
+            Professional-grade rookie cards that appreciate faster than most people's cooking skills and last longer than New Year's resolutions.
+        </div>
+        <div class="card__footer">
+            <button class="btn btn--tertiary btn--fluid">See Details</button>
+        </div>
+    </div>
+</span>
+</highlight-code>
+
+    <h2 id="card-full-hero">Card with Full Hero</h2>
+
+    <p>One of the common layouts for card is to have a full-width hero image that spans the entire width of the card. To accomplish this use the <highlight>.card--full-hero</highlight> modifier.</p>
+
+<demo>
+    <span class="card card--full-hero">
+        <div class="card__media">
+            <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+        </div>
+        <div class="card__main">
+            <div class="card__overline">
+                <svg class="icon icon--16" height="16" width="16">
+                    <use href="#icon-authenticity-guarantee-filled-16-colored" />
+                </svg>
+                <span class="card__overline-text">Authenticity Guaranteed</span>
+            </div>
+            <h3 class="card__title">Authentic Rookie Cards</h3>
+            <div class="card__description">
+                A diversified sports investment portfolio featuring hockey fights, home runs, and three-pointers instead of stocks and bonds.
+            </div>
+            <div class="card__footer">
+                <button class="btn btn--tertiary btn--fluid">See Details</button>
+            </div>
+        </div>
+    </span>
+</demo>
+
+<highlight-code type="html">
+<span class="card card--full-hero">
+    <div class="card__media">
+        <img src=imgCards alt="Card Hero Image" class="card__hero-image">
+    </div>
+    <div class="card__main">
+        <div class="card__overline">
+            <svg class="icon icon--16" height="16" width="16">
+                <use href="#icon-authenticity-guarantee-filled-16-colored" />
+            </svg>
+            <span class="card__overline-text">Authenticity Guaranteed</span>
+        </div>
+        <h3 class="card__title">Authentic Rookie Cards</h3>
+        <div class="card__description">
+            A diversified sports investment portfolio featuring hockey fights, home runs, and three-pointers instead of stocks and bonds.
+        </div>
+        <div class="card__footer">
+            <button class="btn btn--tertiary btn--fluid">See Details</button>
+        </div>
+    </div>
+</span>
+</highlight-code>
+
+    <h2 id="disabled-card">Disabled Card</h2>
+
+    <p>
+        Cards that are disabled will appear grayed out and unclickable. For internally-actionable cards, <highlight>aria-disabled="true"</highlight> attribute needs to be set on the card container. This will ensure that screen readers and other assistive technologies recognize the card as disabled.
```

Comment:
I don't think this will actually do anything, because the root container has no implicit or explicit role.

Reply:
Will remove.